estimagic/dashboard/monitoring_app.py

```diff
@@ -136,10 +157,10 @@ def _create_cds_for_monitoring_app(start_params):
     crit_data = {"iteration": [], "criterion": []}
     criterion_history = ColumnDataSource(crit_data, name="criterion_history_cds")
 
-    param_names = start_params["name"].tolist()
+    param_ids = [id_ for id_list in group_to_param_ids.values() for id_ in id_list]
```

Comment:
Such a nested list comprehension is too complex. Can't we just get the `params_ids` from the `start_params` or if we use integers as IDs just construct it as a range?

Reply:
Would also be possible but the `param_names` are actually not used anymore in this function so I stuck to creating the `param_ids` in a loop from `group_to_param_ids` and got rid of the `start_params` as argument. (0081c97)